sdk/search/azure-search-documents/azure/search/documents/indexes/_search_index_client.py

```diff
@@ -90,7 +90,7 @@ def get_search_client(self, index_name: str, **kwargs: Any) -> SearchClient:
         :rtype: ~azure.search.documents.SearchClient
 
         """
-        return SearchClient(self._endpoint, index_name, self._credential, **kwargs)
+        return SearchClient(self._endpoint, index_name, self._credential, api_version=self._api_version, **kwargs)
```

Comment:
Seems that the `audience` value passed in to `SearchIndexClient` should also be propagated.

Reply:
Good point.

Thanks for catching that!